backend/impl/src/test/resources/version.txt

```diff
@@ -1 +1 @@
-git-machete version 3.15.1
+git-machete version 3.16.0
```

Comment:
Used the version from https://github.com/VirtusLab/git-machete/pull/828/ (yet unreleased)